### PR TITLE
Feature create property

### PR DESCRIPTION
**Pull request by atche1**

### CODE REVIEW

File: src/main/java/com/example/ludogorieSoft/village/config/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SwaggerConfig {
+    @Bean
+    public OpenAPI openAPI() {
+        return new OpenAPI().addSecurityItem(new SecurityRequirement().addList("Bearer Authentication"))
+                .components(new Components().addSecuritySchemes("Bearer Authentication", createAPIKeyScheme()))
+                .info(new Info().title("Football Simulator")
+                        .description("REST API for football simulator."));
```
Comment (Nadezhda358) [inline src/main/java/com/example/ludogorieSoft/village/config/SwaggerConfig.java]: Correct the project name

**Review by Nadezhda358 — CHANGES_REQUESTED** [review]
Remove unnecessary commented-out code

File: src/main/java/com/example/ludogorieSoft/village/services/PropertyService.java [hunk 2]
```diff
@@ -14,26 +13,51 @@
...
-        propertyDTO.setPropertyStatsDTO(propertyStatsDTO);
-        return  propertyDTO;
+
+        return modelMapper.map(property, PropertyDTO.class);
+    }
+    public Property propertyDTOToProperty(PropertyDTO propertyDTO) {
+
+        return modelMapper.map(propertyDTO, Property.class);
     }
+//    private void checkPropertyValidations(PropertyDTO propertyDTO) {
```
Comment (Nadezhda358) [inline src/main/java/com/example/ludogorieSoft/village/services/PropertyService.java]: Uncomment the validation method and fix it if it’s not working

File: src/test/java/com/example/ludogoriesoft/village/services/PropertyServiceTest.java [hunk 3]
```diff
@@ -84,19 +84,19 @@ void whenImageUrlIsValidThenCallsImageServiceAndSetsImageUrl() {
...
-        when(modelMapper.map(property.getPropertyStats(), PropertyStatsDTO.class)).thenReturn(new PropertyStatsDTO());
-
-        PropertyDTO result = propertyService.propertyToPropertyDTO(property);
-
-        assertEquals(propertyDTO, result);
-        verify(modelMapper, times(1)).map(property, PropertyDTO.class);
-        verify(villageService, times(1)).villageToVillageDTO(property.getVillage());
-    }
+//    @Test
+//    void testPropertyToPropertyDTO() {
```
Comment (TodorYanakiev) [inline src/test/java/com/example/ludogoriesoft/village/services/PropertyServiceTest.java]: Delete commented test if not needed

File: src/main/java/com/example/ludogorieSoft/village/services/PropertyService.java (hunk 2 above)
Comment (TodorYanakiev) [inline src/main/java/com/example/ludogorieSoft/village/services/PropertyService.java]: Fix the method if needed

File: src/main/java/com/example/ludogorieSoft/village/config/SwaggerConfig.java (hunk 1 above)
Comment (TodorYanakiev) [inline src/main/java/com/example/ludogorieSoft/village/config/SwaggerConfig.java]: Change title and description

**Review by TodorYanakiev — CHANGES_REQUESTED** [review]

**Pull request merged**